### DISCUSSION TITLE
Opinions & Feedback

### DISCUSSION THREAD

**Discussion by IsmaelMousa** [community/community#114014] [Programming Help] [answered]
### Select Topic Area

Product Feedback

### Body

We value your feedback and suggestions 🤟! If you've used TTL or explored its features, we'd love to hear from you 🫡. share your thoughts, ideas, or any issues you encountered while using the application 💡.

Feel free to open an issue on GitHub. Your feedback helps us improve TTL and make it even better for our users 🎭.

Repo link: https://github.com/IsmaelMousa/TTL

Thank You 🌹🫵,
Ismael.

**Comment by PATRICKNDIZEYE** [comment]
one of the best Todos i have ever seen

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @IsmaelMousa ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
